src/test/java/org/alfresco/repo/search/impl/querymodel/impl/db/DBQueryTest.java

```diff
@@ -899,7 +900,36 @@ public void testAFTS()
         aftsQueryWithCount("=ASPECT:\"test:testSuperAspect\" AND =test:orderMLText:\""+stringValue+"\"", 1);
         
     }
+
+    @Test
+    public void testAftsPagination()

```

Comment:
Please could you add some javadoc explaining roughly what happens in this test?  E.g. "Check that by limiting the maxCount we can get four results returned from a total of six."